addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java

```diff
@@ -374,72 +355,77 @@ public void scan(HttpMessage msg, String param, String value) {
             String phase = null;
```

Comment:
This along with all other unused code should be removed.

Reply:
This is no fully addressed (at least the following are not used: `SLEEP_SHORT_TIME`, `SLEEP_LONG_TIME`, `INSERT_SHORT_TIME`, `INSERT_LONG_TIME`, `aveRtt`, `getAveRtts(List)`, `rtt`, `hadTimeout`).